keps/sig-scheduling/3633-matchlabelkeys-to-podaffinity/README.md

```diff
@@ -758,13 +754,25 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+It shouldn't impact already running workloads. It's an opt-in feature,
+and users need to set `matchLabelKeys` or `mismatchLabelKeys` field in PodAffinity or PodAntiAffinity to use this feature. 
+
+When this feature is disabled by the feature flag, 
+the already created Pod's `matchLabelKeys`/`mismatchLabelKeys` is kept and the `labelSelector` is not modified back.
+But, the newly created Pod's `matchLabelKeys` or `mismatchLabelKeys` field is ignored and silently dropped.
+
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
 
+- A spike on metric `schedule_attempts_total{result="error|unschedulable"}` when pods using this feature are added.
+
+No need to check latency of the scheduler because the scheduler doesn't get changed at all for this feature.
+The only possibility is the bug in the Pod creation process in kube-apiserver and it results in some unintended scheduling.
```

Comment:
Is this really true? The computations on scheduler side might actually be slightly more expensive no? [e.g. a bit larger selectors to compute against pods...]
@alculquicondor - thoughts?

Reply:
Maybe... In some scenarios the users might be migrating from manually setting labels to using the new automated way.

Reply:
> The computations on scheduler side might actually be slightly more expensive no?
> a bit larger selectors to compute against pods

This is correct though, is it realistic that worth being added here?: the labelselector is added by matchLabelKeys → the time the scheduler takes to calculate the labelselector becomes too big. 
If it happens, it's likely not the problem of this feature, but the problem of the entire PodAffinity that labelSelector calculation is somehow buggy.

Reply:
Maybe it's not worth adding, but also please don't mislead the reader that "latency of scheduler doesn't get changed" - it is implicitly changed by using its features more.

Reply:
I added the mention of the scheduler's latency, but with a note that we could care less about that.